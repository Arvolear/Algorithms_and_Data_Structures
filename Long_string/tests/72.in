40939023044526115598524008499226211314838
82026