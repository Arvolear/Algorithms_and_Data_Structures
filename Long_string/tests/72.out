40939023044526115598524008499226211396864
40939023044526115598524008499226211232812
3358064304250299158084530321157529209310901788
499098127965841508771901695794336080